lib/templates/immutable.yaml.mustache

```diff
@@ -19,6 +19,10 @@
       region: '{{=<% %>=}}{{ region }}<%={{ }}=%>'
       aws_zone: '{{=<% %>=}}{{ region }}<%={{ }}=%>{{=<% %>=}}{{ aws_zone }}<%={{ }}=%>'
       group: '{{{ security_group }}}'
+      {{#using_vpc}}
+      vpc_subnet_id: '{{{ vpc_subnet_id }}}'
+      assign_public_ip: yes
```

Comment:
No, not in that case. That's cool, just wanted to check (without doing
research myself!)
On 24 Jun 2016 01:16, "Jason Stillerman" <notifications@github.com> wrote:

> In lib/templates/immutable.yaml.mustache
> <https://github.com/radify/radiian/pull/90#discussion_r68337216>:
>
> > @@ -19,6 +19,10 @@
> >        region: '{{=<% %>=}}{{ region }}<%={{ }}=%>'
> >        aws_zone: '{{=<% %>=}}{{ region }}<%={{ }}=%>{{=<% %>=}}{{ aws_zone }}<%={{ }}=%>'
> >        group: '{{{ security_group }}}'
> > +      {{#using_vpc}}
> > +      vpc_subnet_id: '{{{ vpc_subnet_id }}}'
> > +      assign_public_ip: yes
>
> When using classic, you don't assign a vpc, and assign_public_ip defaults
> to yes. Should assign_public_ip: yes go in anyway?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/radify/radiian/pull/90/files/3105eccb6caac0a38f0e463051707dfa779fa299#r68337216>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAMhvfskYVib_clkZWe3UV_JKy98Oamaks5qOyHugaJpZM4I8OyW>
> .
>